site/content@beta/platform-overview-new/platform-overview.md

```diff
@@ -0,0 +1,80 @@
+![LiveChat Chat Actions](livechat-chat-actions.jpg)
+
+Web appplications can be displayed **fullscreen**, right in the Agent App. If you're planning on integrating your CRM with LiveChat, that might be the way to go.
+
+There's also another area where developers can integrate their apps- it's the Message Box. This interface can be used for extensions related strongly to messages, which are being sent.
+It could be used to send gifs or files from [Dropbox](https://www.livechatinc.com/marketplace/apps/dropbox/).
+
+<div align="center">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/P07QVFOrT04" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+## Chat Widget
+
+Just as there are ways to build upon the Agent App, there are also some to extend the Chat Widget. First of all, you could use [Chat Widget JavaScript API](https://developers.livechatinc.com/docs/js-api/) to alter the look and behavior of the Chat Widget. If that's not enough, building a new Chat Widget from scratch is possible with the [Chat Widget Customer SDK](https://developers.livechatinc.com/docs/visitor-sdk/).
+
+![LiveChat Moments](livechat-moments-in-chat.jpg)
+
+To make chatting more engaging, you could make use of [Chat Widget Moments](https://developers.livechatinc.com/docs/building-apps-for-livechat/#moments) (shown above) and [Rich Messages](https://www.livechatinc.com/help/rich-messages/).
+
+If you want to know more about extending the LiveChat UI, check out [this guide](https://developers.livechatinc.com/docs/building-apps-for-livechat/#extending-user-interfaces).
+
+# Backend APIs
+
+Many applications built upon the LiveChat Platform need to use our API. For that reason, we make it accessible and let developers play with it. 
+
+There are two **messaging APIs**: 
+
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/)
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/)
+
+There's also the [Configuration API](https://developers.livechatinc.com/beta-docs/configuration-api/).
+
+## Chat API
+
+Together, the **Agent Chat API** and the **Customer Chat API** create the **Chat API**. It is a full set of methods to use either as an agent or as a customer. We divide it though, to make it more use-case-specific. The distinction is intuitive; when you want to join a chat as an agent, use the Agent Chat API. To interact as a customer, use the Customer Chat API. 
+
+These two sets of APIs are powerful. The Agent Chat API lets you build your custom Agent App, as well as browser chat archives, or ban customers. With the use of the Customer Chat API, you can build a custom Chat Widget. Both APIs allow for interactions, such as joining a chat, posting messages, etc.
+
+## Configuration API
+
+The **Configuration API** is dedicated to handling licence and agent's properties, webhooks management, and bot agents configuration.
```

Comment:
Configuration API is not really dedicated to handling licence and agent's properties. It's used for storing configuration of license which means creating properties, but possible values for `<property_name>.locations.<location>` while creating a property are: chat, thread or event.

Also, as you've mentioned, Configuration API is used for webhooks and bot agents management (registering  and unregistering webhooks, creating and removing bot agents and so on).